soco/events.py

```diff
@@ -317,11 +317,16 @@ def start(self, any_zone):
         # Sonos net, see http://stackoverflow.com/q/166506
         with self._start_lock:
             if not self.is_running:
-                temp_sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
-                temp_sock.connect((any_zone.ip_address,
-                                   config.EVENT_LISTENER_PORT))
-                ip_address = temp_sock.getsockname()[0]
-                temp_sock.close()
+                if config.EVENT_LISTENER_IP:
```

Comment:
Please add a comment just before this line (We like comments 😄 ):

#Use configured IP address if there is one, else detect automatically